components/bl-clock-component/src/clock-component.js

```diff
@@ -0,0 +1,51 @@
+import { useState, useEffect } from 'react';
+import { getTime } from './helpers';
+import { Time } from './subcomponents';
+
+const TimeVariant = {
+  HHMMSS: 'hhmmss',
+  HHMM  : 'hhmm',
+  HH    : 'hh'
+};
+
+export function ClockComponent({ component }) {
+  const { timeVariant, animationDuration } = component;
+
+  const [time, setTime] = useState(getTime());
```

Comment:
```const [time, setTime] = useState(getTime());```

getTime here will be called every render, you should pass function to useState to prevent it. In this case it will be called only once.